R/helper-functions.R

```diff
@@ -485,35 +548,78 @@ create_sparklines <- function(
       "xr2:uid"           = sprintf("{6F57B887-24F1-C14A-942C-%s}", random_string(length = 12, pattern = "[A-F0-9]"))
     ),
     xml_children = c(
-      xml_node_create("x14:colorSeries",   xml_attributes = color_series),
+      xml_node_create("x14:colorSeries", xml_attributes = color_series),
       xml_node_create("x14:colorNegative", xml_attributes = color_negative),
-      xml_node_create("x14:colorAxis",     xml_attributes = color_axis),
-      xml_node_create("x14:colorMarkers",  xml_attributes = color_markers),
-      xml_node_create("x14:colorFirst",    xml_attributes = color_first),
-      xml_node_create("x14:colorLast",     xml_attributes = color_last),
-      xml_node_create("x14:colorHigh",     xml_attributes = color_high),
-      xml_node_create("x14:colorLow",      xml_attributes = color_low),
+      xml_node_create("x14:colorAxis", xml_attributes = color_axis),
+      xml_node_create("x14:colorMarkers", xml_attributes = color_markers),
+      xml_node_create("x14:colorFirst", xml_attributes = color_first),
+      xml_node_create("x14:colorLast", xml_attributes = color_last),
+      xml_node_create("x14:colorHigh", xml_attributes = color_high),
+      xml_node_create("x14:colorLow", xml_attributes = color_low),
```

Comment:
Please revert the whitespace changes, I actually like the previous code alignment.

Reply:
Fixed. I made sure to stick to your style. But this part slipped through.